src/Sulu/Bundle/MediaBundle/Content/Types/MediaSelectionContentType.php

```diff
@@ -90,7 +90,7 @@ public function read(
         $languageCode,
         $segmentKey
     ) {
-        $data = json_decode($node->getPropertyValueWithDefault($property->getName(), '{}'), true);
+        $data = json_decode($node->getPropertyValueWithDefault($property->getName(), '{"ids": []}'), true);
```

Comment:
@wachterjohannes Can you have a quick look at this commit? The rest of the PR is fine, but this was written by me. It is necessary, because if we leave the default value as it is, then the serializer will write `[]` instead of `{}`, and this will break our administration interface.

Reply:
as long as this is the value if you select no media in the Admin its ok for me :)